### PR TITLE
Add license_id(..) for identifying LICENSE files

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,35 @@ $ identify-cli wat.wat --filename-only; echo $?
...
+The approach that `identify` uses is as follows:
+
+1. Strip the copyright line
+2. Normalize all whitespace
+3. Return any exact matches
+4. Return the closest by edit distance (where edit distance < 5%)
+
+To use the api, install via `pip install identify[license]`
+
+```pycon
```
Comment (chriskuehl) [inline README.md]: pycon? :D

File: bin/vendor-licenses [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""Usage:
+
+    ./bin/vendor-licenses > identify/vendor/licenses.py
+"""
+from __future__ import absolute_import
+from __future__ import unicode_literals
```
Comment (chriskuehl) [inline bin/vendor-licenses]: could remove these imports + exclude this from the hook i guess /shrug

File: README.md (hunk 1 above)
Comment (asottile) [reply README.md]: yeah, there's a pygments lexer for "python console" :D

File: bin/vendor-licenses (hunk 2 above)
Comment (asottile) [reply bin/vendor-licenses]: yeah, figured it doesn't hurt -- it's syntax compatible with python2, just not runtime compatible

**Pull request merged**